_component_design/tooltip.md

```diff
@@ -13,28 +13,304 @@ last-modified: 2017-08-17
 {% endfigure %}
+
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Tooltip specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Tooltips/tooltip-specs.svg){:width="100%"}
+</div>
+
+## Triggers
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Tooltips will only appear after be activated by a trigger. Triggers come in a variety of types.
+
+### Trigger types
+{: .hxSubSectionTitle}
+
+- **Icon:** Use this trigger variation when a recognizable icon on the application informs a user that additional information is available for consumption if needed. (e.g. A ‘?’ for help or ‘$’ for pricing) 
+- **Table text:** This variation is used to reveal additional information within a table. By hiding information behind tooltips in a table we keep the tables easily consumable and only display additional information when a user requires. Do not hide important information behind a tooltip. Note: May apply to both plain text cells and link text cells.
+- **Data visualization:** Use this trigger variation to present robust data sets aggregated into a data visualization.
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Examples of tooltips generated from various trigger types."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-triggers.svg){:width="100%"}
+
 {% endfigure %}
 {% endcolumn %}
 </div>
+
+## Text conventions
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{: .hxSubSectionTitle}
+
+- To keep the tooltip text concise, you should use short sentences and sentence fragments. 
+- Try to keep your tooltips under three lines of text including a link if you need one.
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Most tooltips should be very short and only provide enough information for the user to better understand the context. Tooltips can also be used to reveal information that is truncated or hidden by default."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-text-conventions-1.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Tooltip variations
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Help text with a link
+{: .hxSubSectionTitle}
+
+This variation is used when a tooltip need to provide access to a singular external webpage.
+
+- This link should always be an external link and should open in a new tab. 
+- Be sure you always include an external link icon
+- If you need additional inputs or interactions on a tool tip, e.g. a dropdown or selector, use the larger popover pattern.
```

Comment:
tooltip, for example, a drop-down or selector, use the larger tooltip pattern.*

Reply:
updated.